.github/ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,9 @@ Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedChe
 /var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
 #[[PLACE YOUR OUTPUT HERE]]
 ```
+For Windows users, please use `type` instead of `cat` and run 
+`java -Duser.language=en -Duser.country=US -jar checkstyle-X.XX-all.jar
+-c config.xml YOUR_FILE.java` in place of the last command above.
```

Comment:
If I remember it correctly, this should be something like that
```cmd
D:\> SET RUN_LOCALE="-Duser.language=en -Duser.country=US"
D:\> java %RUN_LOCALE% -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
```

Reply:
I can confirm that it works, revised the template